src/shared/components/FormField/FormField.style.ts

```diff
@@ -12,14 +12,13 @@ export const FormFieldWrapper = styled.div`
 
 export const FormFieldHeader = styled.header`
   display: flex;
-  align-items: center;
+  align-items: baseline;
```

Comment:
This will actually cause the header to have different height based on whether or not it is optional. Had a chat with designers about this, conclusion is we'll try to figure out this font issue in general. For now let's just use `center` here and accept non-perfect look. Hopefully we resolve the font issue soon and it just fixes itself